standards/countries.rst

```diff
@@ -0,0 +1,44 @@
+.. _countries:
+
+A common list of countries
+==========================
+
+Having a standardized list of country names is an important prerequisite for region
+definitions, scenario analysis and model comparison.
+
+The :class:`nomenclature` package (`read the docs`_) builds on the :class:`pycountry`
+package to provide a standardized list if country names based on the
```

Comment:
```suggestion
package to provide a standardized list of country names based on the
```

EDIT: Having read this paragraph further, you do link the full set of changes, below. I'd put it here, maybe in addition to below.